### ISSUE TITLE
Rename tgstation.dmm to boxstation.dmm or similar.

### ISSUE THREAD

**Issue by PJB3005** [d3athrow/vgstation13#28536]
TG doesn't even have box anymore, the heretics.

I propose we fix this inaccuracy to show our independence from TGstation.

**Comment by DeityLink** [comment]
Seems like as good a time as any.

**Comment by PJB3005** [comment]
![](https://i.ytimg.com/vi/_T2c8g6Zuq8/maxresdefault.jpg)

**Comment by DeityLink** [comment]
Yeah well I was told it'd cause some issues with filepaths and stuff so I figured I'd let @d3athrow do it if it's really worth it.
